app/helpers/users_helper.rb

```diff
@@ -0,0 +1,5 @@
+module UsersHelper
+  def user_profile_image_link(user)
+    current_user.profile_image.present? ? rails_blob_path(current_user.profile_image, disposition: 'attachment') : 'default.jpeg'
```

Comment:
Will 'default.jpeg' only will work? Don't you need exact path?
Is this already in repo?

Reply:
Yes, already in repo in assets folder
[Image](https://github.com/siddharth-vinsol/quora-clone/blob/feature/user-profile/app/assets/images/default.jpeg)